guide/yaml/test/test-entities.md

```diff
@@ -89,6 +93,10 @@ The `TestEffector` entity invokes the specified effector on a target entity. If
 - `params` - parameters to pass to the effector, these will depend on the entity and effector being tested. The example above shows the `url` and `targetName` parameters being passed to Tomcats `deploy` effector.
 - `assert` - assertion to perform on the returned result. See section on assertions below.
 
+<div class="alert alert-info">
```

Comment:
The `include` seems nice but the line will become quite long depending on the text to pass right?

As for using HTML over md, I didn't know I could use markdown inside HTML so quite happy to change if you prefer.